composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
The goal is to prevent packages that depend on `equip/data` from having conflicts with this package.